i18n/ads-language-pack-de/package.json

```diff
@@ -215,6 +162,58 @@
 					{
 						"id": "vscode.yaml",
 						"path": "./translations/extensions/yaml.i18n.json"
+					},
+					{
+						"id": "Microsoft.admin-tool-ext-win",
```

Comment:
This shouldn't be including extensions that aren't bundled into the main product right?

Reply:
This was already an existing translation that was in earlier lang-packs, I will remove the entry and file for this and other external extensions